README.md

```diff
@@ -8,6 +8,17 @@ This library provides RSpec matcher for testing JSON string.
 * handy pattern matching with `#===` method like case-when statement
 * supporting nested pattern matching
 
+## Installation
+
+Install RSpec::JsonMatcher:
+```shell
+$ gem install rspec-json_matcher
+```
+Or add it to your Gemfile:
+```ruby
+gem 'rspec-json_matcher'
+```
+
```

Comment:
Thank you. For what it's worth, could you please follow the format that the latest bundler outputs?

```suggestion
## Installation

Install the gem and add to the application's Gemfile by executing:

    $ bundle add rspec-json_matcher

If bundler is not being used to manage dependencies, install the gem by executing:

    $ gem install rspec-json_matcher
```

BTW, it is incorrect to use the `$` sign while specifying `shell` for language. If you want to use `$`, you should specify `console` for language.